scripts/live-demo.py

```diff
@@ -103,7 +106,12 @@ def main(camera_id, filename, hrnet_c, hrnet_j, hrnet_weights, hrnet_joints_set,
             frame = draw_points_and_skeleton(frame, pt, joints_dict()[hrnet_joints_set]['skeleton'], person_index=pid,
                                              points_color_palette='gist_rainbow', skeleton_color_palette='jet',
                                              points_palette_samples=10)
+            if extract_kpts:
+                dic[frame_count] = pt[:, :2]
+            # print("\nframe: {}".format(frame_count))
 
+        if extract_kpts:
+            np.savez("output_kpts", dic)
```

Comment:
This should be moved outside the while loop. Here it slows down the performance and does a lot of unnecessary writes.

Reply:
You can also consider to use savez_compressed in place of savez